examples/preprocessing/plot_artifact_correction_with_faster.py

```diff
@@ -0,0 +1,93 @@
+"""
+====================================
+EEG artifact correction using FASTER
+====================================
+
+In this example, a variety of metrics are used to detect channels and epochs
+that contain artifacts. Rejection and interpolation are used to clean the EEG
+data.
+
+References
+----------
+[1] Nolan H., Whelan R. and Reilly RB. FASTER: fully automated
+    statistical thresholding for EEG artifact rejection. Journal of
+    Neuroscience Methods, vol. 192, issue 1, pp. 152-162, 2010.
+"""
+import mne
+from mne import io
+from mne.datasets import sample
+
+from mne_sandbox.preprocessing import (find_bad_channels, find_bad_epochs,
+                                       find_bad_channels_in_epochs)
```

Comment:
I feel that if all these find_* commands are faster based we should use a faster namespace. So I would write:

from mne_sandbox.preprocessing import faster
faster. find_bad_channels(...)

it would make it explicit what is used and allow to use different approaches in the future.